src/sections/hero.html

```diff
@@ -0,0 +1,47 @@
+<section
+  class="h-cg-s lg:h-cg-l lg:max-w-cg-l overflow-hidden flex lg:relative lg:items-end lg:mt-8 lg:mb-64 my-14 px-5"
+>
+  <div
```

Comment:
Please install the class name order extension. This extension helps to order the tailwind classes which made code base easy to read. 

[Classname order package](https://tailwindcss.com/blog/automatic-class-sorting-with-prettier)